gluoncv/model_zoo/danet.py

```diff
@@ -0,0 +1,201 @@
+"""Dual Attention Network"""
+from mxnet.gluon import nn
+from mxnet.context import cpu
+from mxnet.gluon.nn import HybridBlock
+from .segbase import SegBaseModel
+from .fcn import _FCNHead
+from .attention import PAM_Module, CAM_Module
+# pylint: disable-all
+
+__all__ = ['DANet', 'get_danet', 'get_danet_resnet50_citys', 'get_danet_resnet101_citys']
+
+class DANet(SegBaseModel):
+    r"""Dual Attention Networks for Semantic Segmentation
+    Parameters
+    ----------
+    nclass : int
+        Number of categories for the training dataset.
+    backbone : string
+        Pre-trained dilated backbone network type (default:'resnet50'; 'resnet50',
+        'resnet101').
+    norm_layer : object
+        Normalization layer used in backbone network (default: :class:`mxnet.gluon.nn.BatchNorm`;
+    Reference:
+        Jun Fu, Jing Liu, Haijie Tian, Yong Li, Yongjun Bao, Zhiwei Fang, Hanqing Lu. "Dual Attention 
+        Network for Scene Segmentation." *CVPR*, 2019
+    """
+
+    def __init__(self, nclass, backbone='resnet50', aux=False, ctx=cpu(), pretrained_base=True,
+                 height=None, width=None, base_size=576, crop_size=512, dilated=True, **kwargs):
```

Comment:
Why base_size 576 and crop_size 512? It is kind of strange, because for ADE, the default is 480 and for cityscapes, the default is 768.